src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -439,9 +451,19 @@ private void testByConfig(String configPath, String inputFile, String[] expected
 
         // run RootModule
         final String path = getPath(inputFile);
-        final List<File> files = Collections.singletonList(
-                new File(path));
-        final int errorCounter = rootModule.process(files);
+        final File file = new File(path);
+        final int errorCounter;
+        if (file.isDirectory()) {
+            final File[] files = new File(path).listFiles();
+            final List<File> theFiles = new ArrayList<>();
+            Collections.addAll(theFiles, files);
+            errorCounter = rootModule.process(theFiles);
+        }
+        else {
+            final List<File> files = Collections.singletonList(
+                    new File(path));
+            errorCounter = rootModule.process(files);
```

Comment:
Same line as in the if condition. Please move it out.

Reply:
done